src/main/java/anab/Person/Person.java

```diff
@@ -22,28 +38,51 @@ public void setLastName(String newLastName) {
     }
 
 
-    public String getAge() {
-        return Age;
+    public Integer getAge() {
+        return age;
     }
 
-    public void setAge(String newAge) {
-        Age = newAge;
+    public void setAge(Integer newAge) {
+        age = newAge;
     }
 
 
     public String getGender() {
-        return Gender;
+        return gender;
     }
 
     public void setGender(String newGender) {
-        Gender = newGender;
+
+        //4. Add validation to setGender(String gender) allow setting the gender as “Male”, “Female“, “Other”.
+        if ((newGender == "Male") || (newGender == "Female") || (newGender == "Other")) {
+            gender = newGender; //assign value given checked by if conditions
+            //  System.out.println("Gender value is: " + gender); - not needed since we only wanna assign the value and display it using main method
```

Comment:
same here, please delete this commented line

Reply:
Deleted commented line: //4. Add validation .....